src/omv/modules/py_tf.c

```diff
@@ -596,9 +503,13 @@ STATIC mp_obj_t py_tf_segment(uint n_args, const mp_obj_t *pos_args, mp_map_t *k
     py_tf_model_obj_t *model = py_tf_load_alloc(pos_args[0]);
     uint8_t *tensor_arena = fb_alloc(model->params.tensor_arena_size, FB_ALLOC_PREFER_SPEED | FB_ALLOC_CACHE_ALIGN);
 
-    py_tf_input_callback_data_t py_tf_input_callback_data;
+    py_tf_input_callback_data_t py_tf_input_callback_data =
+    {.input_normalization = {0.0f, 0.0f, 0.0f, 1.0f, 1.0f, 1.0f}};
     py_tf_input_callback_data.img = image;
     py_tf_input_callback_data.roi = &roi;
+    py_tf_input_callback_data.input_scale = args[ARG_input_scale].u_int;
+    py_helper_arg_to_float_array(args[ARG_input_normalization].u_obj,
+                                 py_tf_input_callback_data.input_normalization, PY_TF_INPUT_NORMALIZATION_SIZE);
```

Comment:
```suggestion
```suggestion
    py_tf_input_callback_data_t py_tf_input_callback_data = {
        .img = image,
        .roi = &roi,
        .input_scale = args[ARG_input_scale].u_int,
        .input_stats = {0.0f, 0.0f, 0.0f, 1.0f, 1.0f, 1.0f},
    };
    if (args[ARG_input_stats].u_obj != mp_const_none) {
        py_helper_arg_to_float_array(args[ARG_input_stats].u_obj, py_tf_input_callback_data.input_stats, PY_TF_INPUT_STATS_SIZE);
    }
```

Reply:
Done

Note: py_helper_arg_to_float_array does "if (args[ARG_input_stats].u_obj != mp_const_none)" internally.